core/runtime/src/main/java/org/apache/isis/core/runtime/system/persistence/PersistenceSessionBase.java

```diff
@@ -134,16 +128,12 @@ protected PersistenceSessionBase(
         this.userService = lookupService(UserService.class);
 
         // sub-components
-        final AdapterManager adapterManager = this;
-        this.persistenceQueryFactory = new PersistenceQueryFactory(adapterManager, this.specificationLoader);
+        this.persistenceQueryFactory = new PersistenceQueryFactory(
+                obj->this.getObjectAdapterProvider().adapterFor(obj), 
+                this.specificationLoader);
         this.transactionManager = new IsisTransactionManager(this, /*authenticationSession,*/ servicesInjector);
 
         this.state = State.NOT_INITIALIZED;
-
```

Comment:
Did this configuration property get removed?  Or was the code just moved elsewhere?  I'd prefer to keep it than lose it.

Reply:
regarding property 'concurrencyCheckingGloballyEnabled': code moved here

https://github.com/apache/isis/blob/2c7041016faf87443befaa80eef9a0457ac1338f/core/metamodel/src/main/java/org/apache/isis/core/metamodel/adapter/concurrency/ConcurrencyChecking.java#L32